src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitTransactionStorage = new HashMap<>();
+
+    public static int getQuantity(String fruit) {
+        return fruitTransactionStorage.entrySet().stream()
+                .filter(k -> k.getKey().equals(fruit))
```

Comment:
avoid short one-letter naming in stream chains outside of math context
change to `fruitKey` or `fruitName`